renderer/components/Settings/SettingsPage.js

```diff
@@ -108,6 +125,7 @@ const SettingsPage = ({ currentConfig, ...rest }) => {
         <SettingsForm>
           {group === 'general' && <SettingsFieldsGeneral currentConfig={currentConfig} />}
           {group === 'wallet' && <SettingsFieldsWallet currentConfig={currentConfig} />}
+          {group === 'security' && <SettingsFieldsSecurity currentConfig={currentConfig} />}
```

Comment:
since this list keeps growing and all components share the same prop list, consider converting this into a map of `<group, component>` to reduce boilerplate code.